src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -201,6 +203,10 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.hasChromiumWebRTC() || this.isSafari();
   }
 
+  doesNotSupportMediaDeviceLabels(): boolean {
```

Comment:
Can you leave a comment above this with the link to the chromium bug

Reply:
Added!